### PR TITLE
feat(wokwi-hx711): add docs for HX711 Load Cell Amplifier

### PR DESCRIPTION
**Pull request by ishotjr**

### CODE REVIEW

File: docs/parts/wokwi-hx711.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Attributes
+
+| Name     | Description                                | Default value |
+| -------- | ------------------------------------------ | ------------- |
+| type     | Either "50kg" (default), "5kg", or "guage" | "50kg"        |
+
+## Operation
+
+The HX711 amplifier allows you to easily read load cells and evaluate changes in resistance. A Wheatstone bridge is used to connect load cells to the IC, which is in turn connected to the microcontroller via `VCC`, `DT`, `SCK`, and `GND`. Use `begin()` to initialize the scale and `set_scale()` and `tare()` to calibrate it. `power_down()` and `power_up()` can be used to bring the ADC into and out of low power mode. `get_value()` and `get_units()` are used to read the ADC minus tare and divided, passing an optional integer value to obtain that number of values, averaged.
```
Comment (urish) [inline docs/parts/wokwi-hx711.md:31]: It's worth mentioning that all these methods are part of the `HX711` library for Arduino (I can see there are several Arduino libraries for interfacing with the HX711).

File: docs/parts/wokwi-hx711.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| Name | Description                                       |
+| ---- | ------------------------------------------------- |
+| VCC  | Voltage supply (5V)                               |
+| DT   | Serial data                                       |
+| SCK  | Serial clock                                      |
+| GND  | Ground                                            |
+
+*note that `E+`/`E-`/`A+`/`A-`/`B+`/`B-` pins are non-interactive and rendered based on attributes*  
+
+<wokwi-hx711 type="5kg" />
```
Comment (urish) [inline docs/parts/wokwi-hx711.md:21]: We usually create a table at the bottom of the "Attributes" section with examples of how the attributes affect the visual style of the element, e.g. [here](https://docs.wokwi.com/parts/wokwi-7segment#examples)

**Review by urish — CHANGES_REQUESTED** [review]
Thanks! I added some comments, also - please add this to sidebar.js (this is the file that controls the documentation ToC)

**Comment by urish** [comment]
Thanks! A few more things:

1. There's a typo: guage -> gauge (note that the 3rd example in the "Examples" table is the same as the 2nd example, because of the typo).
2. Can we simplify the example in the docs? Ideally, the example would be just the minimal thing the user need to get some input from the sensor, and then we can have links to more elaborate examples in the "Simulator examples" section.
3. Can you please format the markdown file with prettier, to keep consistent with the other files? There's a prettier extension for VSCode, but you can also type `npx prettier --write docs` in the terminal to reformat everything (just don't commit unrelated files)

**Comment by ishotjr** [comment]
@urish that's so strange RE `guage` - I pasted the whole thing into Word to check for typos before submitting the PR! 😅 Requested changes are in, please let me know if there's anything else! :bowtie:

**Pull request merged**